src/pages/guidelines/spacing.mdx

```diff
@@ -20,9 +20,9 @@ Carbon for IBM.com uses uniform components, elements, and spacing to encourage c
 
 ## Designing with space
 
-Every part of a UI should be intentional, including the empty space, also known as white space, between elements. The amount of space between items creates relationships and hierarchy. You can learn more about white space and how space can help you create relationships and hierarchy on the [Carbon spacing guideline page](https://www.carbondesignsystem.com/guidelines/spacing/overview/#designing-with-space).
+Every part of a UI should be intentional, including the empty space, also known as white space, between elements. The amount of space between items creates relationships and hierarchy. You can learn more about white space and how space can help you create relationships and hierarchy on the [Carbon spacing guidelines](https://www.carbondesignsystem.com/guidelines/spacing/overview/#designing-with-space) page.
 
-When designing for IBM.com, the spacing scale and type tokens are used to ensure the information is well organized with consistency.
+When designing for IBM.com, the spacing scale and type tokens are used to ensure the information is well organized with consistency, see the [Type pairings guidance](../expressive-styling/type-pairing) to learn more about the type hierarchy of the Carbon for IBM.com components.
```

Comment:
```suggestion
When designing for IBM.com, the spacing scale and type tokens are used to ensure the information is well organized with consistency. See the [Type pairings guidance](../expressive-styling/type-pairing) to learn more about the type hierarchy of the Carbon for IBM.com components.
```

I feel like it's a little long of a sentence, just adding a period to break it up. Everything else looks good!